components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -89,6 +93,14 @@
 public class IdPManagementDAO {
 
     private static final Log log = LogFactory.getLog(IdPManagementDAO.class);
+    private static final String secretType = "FEDERATED_AUTHENTICATOR";
```

Comment:
based on the pending improvements to the secret type in the secret management component this might change.